packages/grafana-ui/src/components/DataSourceSettings/SigV4AuthSettings.tsx

```diff
@@ -97,21 +97,17 @@ export const SigV4AuthSettings: React.FC<HttpSettingsProps> = props => {
             <Select
               className="width-30"
               value={authProviderOptions.find(
-                authProvider => authProvider.value === dataSourceConfig.jsonData.authType
+                authProvider => authProvider.value === dataSourceConfig.jsonData.sigV4AuthType
               )}
               options={authProviderOptions}
-              defaultValue={dataSourceConfig.jsonData.authType || 'keys'}
+              defaultValue={dataSourceConfig.jsonData.sigV4AuthType || 'keys'}
```

Comment:
Fallback to default doesn't seem to work:
![image](https://user-images.githubusercontent.com/1668778/95743861-b6689d80-0c92-11eb-8be1-cdcc6cf65f74.png)

I also think that default should be "default"/AWS SDK Default

Reply:
Weird.. must have got messed up from switching from the original design to the existing CW config editor. Looks like that's even changed again so we might be able to just make them consistent presuming CW works as expected.

Agree regarding the default 👍

Reply:
Okay I'll merge this one for now and handle changes in upcoming PR when CloudWatch auth changes have been merged